docs/manage/reference/viam-agent/manage-viam-agent.md

```diff
@@ -50,6 +53,60 @@ aliases:
 
   This command uninstalls `viam-agent`, `viam-server`, the machine cloud credentials file (<file>/etc/viam.json</file>), and the provisioning configuration file (<file>/etc/viam-provisioning.json</file>).
 
+{{% /tab %}}
+{{% tab name="Windows native" %}}
+
+On Windows, you can manage `viam-agent` using the Services GUI or the command line.
+
+## Manage `viam-agent` using the Services GUI
+
+1. Open the **Services** management console from your computer's start menu.
```

Comment:
[opt] reverse the order of this phrasing so readers don't have to double back:

```suggestion
1. In your computer's start menu, open the **Services** management console.
```

Disclaimer: I haven't used Windows since 10, so maybe the original order was more natural?

Reply:
In general I'd go with your wording because if someone needs to go to a page before finding a button, it's better to say it in that order instead of "find the x button on the page you didn't know until now you should be on". But in this case I prefer it this way because they don't need to look for something that says "start"--they just need to find "Services" in the natural way, so the emphasis can be there.